docs/es/README.md

```diff
@@ -0,0 +1,13 @@
+# Algoritmos
+
+## Ordenamiento
+
+- [Quick Sort](./Ordenamiento/Quick-Sort.md)
+
+## Cadenas
+
+## Búsqueda
+
```

Comment:
You could remove these due to we don't have the documentation about theses sections.
```suggestion
```